File: Digital Business Enablement/bluegather/src/main/java/br/com/fiap/bluegather/dto/UsuarioDTO.java
package br.com.fiap.bluegather.dto;

import jakarta.validation.constraints.*;

import lombok.*;

import java.util.Set;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UsuarioDTO {
    private Long id;

    @NotNull(message = "O campo cpf não pode estar vazio.")
    private String cpf;

    @NotNull(message = "O campo nome não pode estar vazio.")
    private String nome;

    private String urlImagem;

    @Email(message = "Endereço de e-mail inválido.")
    @NotBlank(message = "O campo email não pode estar vazio.")
    private String email;

    @NotBlank(message = "O campo senha não pode estar vazio.")
    private String senha;

    private Set<Long> idsEventos;        
}

File: Enterprise Application Development/bluegather/Dtos/UsuarioDto.cs
using System.ComponentModel.DataAnnotations;

namespace BlueGather.Dto
{
    public class UsuarioDto
    {
        public long? Id { get; set; }

        [Required(ErrorMessage = "O campo cpf não pode estar vazio.")]
        public string Cpf { get; set; }

        [Required(ErrorMessage = "O campo nome não pode estar vazio.")]
        public string Nome { get; set; }

        public string UrlImagem { get; set; }

        [EmailAddress(ErrorMessage = "Endereço de e-mail inválido.")]
        [Required(ErrorMessage = "O campo email não pode estar vazio.")]
        public string Email { get; set; }

        [Required(ErrorMessage = "O campo senha não pode estar vazio.")]
        public string Senha { get; set; }

        public List<long>? IdsEventos { get; set; }
    }
}